packages/neon/neon/lib/src/widgets/resolve_sync_conflicts_dialog.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:neon/l10n/localizations.dart';
+import 'package:neon/src/sync/conflicts.dart';
+import 'package:neon/src/theme/dialog.dart';
+import 'package:synchronize/synchronize.dart';
+
+class NeonResolveSyncConflictsDialog<T1, T2> extends StatefulWidget {
+  const NeonResolveSyncConflictsDialog({
+    required this.conflicts,
+    super.key,
+  });
+
+  final SyncConflicts<T1, T2> conflicts;
+
+  @override
+  State<NeonResolveSyncConflictsDialog<T1, T2>> createState() => _NeonResolveSyncConflictsDialogState<T1, T2>();
+}
+
+class _NeonResolveSyncConflictsDialogState<T1, T2> extends State<NeonResolveSyncConflictsDialog<T1, T2>> {
+  var _index = 0;
+  final _solutions = <String, SyncConflictSolution>{};
+
+  SyncConflict<T1, T2> get conflict => widget.conflicts.conflicts[_index];
+
+  SyncConflictSolution? get selectedSolution => _solutions[conflict.id];
+
+  void onSolution(final SyncConflictSolution solution) {
+    setState(() {
+      _solutions[conflict.id] = solution;
+    });
+  }
+
+  @override
+  Widget build(final BuildContext context) {
+    final body = Column(
+      children: [
+        Text(
+          AppLocalizations.of(context).syncResolveConflictsTitle(
+            widget.conflicts.conflicts.length,
+            AppLocalizations.of(context).appImplementationName(widget.conflicts.implementation.appId),
+          ),
+          style: Theme.of(context).textTheme.headlineMedium,
+        ),
+        const Divider(),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            _buildConflictCard(
+              title: AppLocalizations.of(context).syncResolveConflictsLocal,
+              solution: SyncConflictSolution.overwriteA,
+              child: widget.conflicts.implementation.getConflictDetailsLocal(context, conflict.objectB.data),
+            ),
+            _buildConflictCard(
+              title: AppLocalizations.of(context).syncResolveConflictsRemote,
+              solution: SyncConflictSolution.overwriteB,
+              child: widget.conflicts.implementation.getConflictDetailsRemote(context, conflict.objectA.data),
+            ),
+          ],
+        ),
+        const Divider(),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            if (_index > 0) ...[
+              OutlinedButton(
+                onPressed: () {
+                  setState(() {
+                    _index--;
+                  });
+                },
+                child: Text(AppLocalizations.of(context).actionPrevious),
+              ),
+            ] else ...[
+              OutlinedButton(
+                onPressed: () {
+                  Navigator.of(context).pop();
+                },
+                child: Text(AppLocalizations.of(context).actionCancel),
+              ),
+            ],
```

Comment:
```suggestion
bool get isLast => index == widget.conflicts.conflicts.length;
bool get isFirst => index == 0;

void movePrevious() {
  if (isFirst){
    if (_solutions.length != widget.conflicts.conflicts.length) {
      return;
    }
    Navigator.of(context).pop(_solutions);
  } else {
    setState(() {
      index--;
    });
  }
}

[ ... ]

              OutlinedButton(
                onPressed: movePrevious,
                child: Text( isFirst? 
                  AppLocalizations.of(context).actionCancel
                  : AppLocalizations.of(context).actionPrevious,
                ),
              ),
```

I hope this is understandable :)